spec/views/posts_index_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+# rubocop:disable Metrics/BlockLength
+RSpec.feature 'PostIndex', type: :feature do
+  let(:user) do
+    User.create(
+      name: 'Fatima',
+      photo: 'https://www.img2link.com/images/2023/04/13/c2bbea766ec481f3d798809dd39eedb6.png',
+      bio: 'CEO Nairobi Hub', posts_counter: 2
+    )
+  end
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/233175747-98586ea6-32f4-4c8e-a9b6-1feca7b5651d.png)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages more meaningful and capitalize 👍 because some of your commit messages are not capitalized 😨 Please consider this issue to make your pull request professional and descriptive ☺️